deploy/flux-installer/templates/apps/istio/istio-ingress-repo.yaml

```diff
@@ -0,0 +1,12 @@
+---
```

Comment:
This is a duplicate of the istio base repo, no need to define the repo twice. You can use a single repo for multiple helm releases

Reply:
I had to create multiple HRs to define the name for base, istiod and gateway. I will work with it to see if I can get it down to 1 HR.

Reply:
Removed istio-ingress-repo.